src/components/home.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { filterData, getData } from '../redux/home/home';
+import pointer from '../images/assets/pointer.svg';
+
+const Home = () => {
+  const data = useSelector((data) => data.detailDataReducer);
+  const onLoad = useSelector((data) => data.loadingStateReducer);
+  const dispatch = useDispatch();
+  const mainArray = ['AAPL', 'FB', 'INTC', 'ORCL', 'NKE', 'PFE', 'NOK', 'TWTR', 'CSCO', 'BABA', 'ATVI', 'FOXA'];
+  const [filterState, setFilterState] = useState(0);
+  useEffect(() => {
+    dispatch(getData(mainArray));
+  }, []);
+
+  const handleClick = (props) => {
+    dispatch(filterData(props));
+  };
+
+  const handleChange = () => {
+    const filterDis = document.getElementById('filterDisplay');
+    setFilterState(filterDis.value);
+  };
+
+  return (
+    <div>
+      <div className="Home_Header_1">
+        <h1>Cripto Up to date</h1>
+        <h2>12 results</h2>
+      </div>
+      <div className="Home_Header_2">
+        <h1>STATS BY COMPANY</h1>
+        <select id="filterDisplay" className="filter" onChange={() => { handleChange(); }}>
+          <option disabled selected value="0">Filter by Value</option>
+          <option value="0">All</option>
+          <option value="50">50+</option>
+          <option value="100">100+</option>
+          <option value="150">150+</option>
+          <option value="200">200+</option>
+          <option value="250">250+</option>
+        </select>
+      </div>
```

Comment:
- [ ] The filter should be a component on its own and it should be stateless. Kindly create a separate filter component and call it in the home component as per the project requirements